qiskit/primitives/estimator.py

```diff
@@ -89,6 +89,27 @@ def __init__(
         )
         self._is_closed = False
 
+    def __repr__(self):
+        circ_names = []
+        for circuit in self._circuits:
+            circ_names.append(circuit.name)
+        param_counts = []
+        for param in self._parameters:
+            param_counts.append(str(len(param)))
+        circ_names = "','".join(circ_names)
+        param_counts = ",".join(param_counts)
+        observables = []
+        for observation in self._observables:
+            observables.append(repr(observation))
+        observables = ",".join(observables)
+        observables = " ".join(observables.split()).replace("\n", "")
+        return (
+            f"< {type(self).__name__} with "
```

Comment:
Same comment with Jake (extra space around `<` and `>`)

Passing circuits and observables during initialization is now deprecated. Therefore, it is better not to display them here as well and just display `_run_options`.